app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,83 @@
+class EmployeesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_employee, only: %i[ new edit update destroy ]
+  before_action :project_manager_only, only: %i[ new create ]
+
+  # GET /employees or /employees.json
+  def index
+    @employees = Employee.all
+  end
+
+  # GET /employees/1 or /employees/1.json
+  def show
+    @employee = Employee.includes(:tasks).find(params[:id])
```

Comment:
Skip the `before_action` in certain cases to avoid N+1 queries in the view.  There are several other examples like this all with similar motivation